src/controllers/UserController.ts

```diff
@@ -29,6 +30,19 @@ export class UserController {
 
     return { role: queriedRoleFromId };
   }
+
+  @Get('/:userID')
+  @ResponseSchema(AppUserProfileResponse)
+  async getUserProfile(
+    @Param('userID') userID: number,
+    @CurrentUser({ required: true }) appUser: AppUser
+  ): Promise<AppUserProfileResponse | undefined> {
+    const { id } = appUser;
+
+    const appUserObj: AppUser = await this.appUserService.getAppUserById(userID);
```

Comment:
Not sure what behavior we're trying to achieve here, but if it's something along the lines of "members and inductees can only look up their own user profile using their own id, officers can look up user profiles using any user's existing id" then you can probably use the code I wrote for getUserRole as reference.

Reply:
The issue description stated authorization checks to be a non-goal. However, I can work on this if that has changed. @godwinpang @thaigillespie

Reply:
yeah @thaigillespie it's fine if we address auth in a separate pr now that we have dev and prod separate :)

Reply:
All the infrastructure for auth and stuff is there as well if you want to try your hands at it @AneeshJalan

Reply:
This will be done in a separate PR/Issue.